docs/docsite/known_good_reqs.txt

```diff
@@ -13,5 +13,5 @@ rstcheck==3.3.1
 sphinx==4.0.2
 sphinx-notfound-page==0.7.1 # must be >= 0.6
 sphinx-intl==2.0.1
-sphinx_ansible_theme===0.6.0
+sphinx-ansible-theme===0.7.0
```

Comment:
@acozine I normalized the name the same way PyPI does — it replaces underscores with dashes.

Reply:
Yes, that makes sense. I will type it wrong 50% of the time, but it's the correct way forward.